client/src/pages/EpisodeShowPage/index.tsx

```diff
@@ -397,13 +424,13 @@ export class WorkspaceView extends React.Component<any, any> {
         snapshot: JSON.stringify({
           userId: Auth.userId(),
           workspaceId: workspace.id,
-          workspace: reduxBlocks.filter(b =>
+          workspace: augmentedBlocks.filter(b =>
             workspace.blocks.find(b2 => b.id === b2.id),
           ),
           children: _.sortBy(workspace.childWorkspaces, cw =>
             Date.parse(cw.createdAt),
           ).map(w => {
-            const childBlocks = reduxBlocks.filter(b =>
```

Comment:
Noticed that the code twice creates the `snapshot` object, once to console.log it and once to props.createSnapshot. Could you change this into one object creation that you pass into the two functions?

Reply:
After thinking about it I think it's fine to get rid of the `console.log`.